crypto/cipher/e_des.cc

```
@@ -50,22 +54,24 @@ static const EVP_CIPHER evp_des_cbc = {
+    /*update_aad=*/nullptr,
     /*cleanup=*/nullptr,
     /*ctrl=*/nullptr,
 };
 
-const EVP_CIPHER *EVP_des_cbc(void) { return &evp_des_cbc; }
+const EVP_CIPHER *EVP_des_cbc() { return &evp_des_cbc; }
 
-static int des_ecb_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out, const uint8_t *in,
-                          size_t in_len) {
-  if (in_len < ctx->cipher->block_size) {
+static int des_ecb_cipher_update(EVP_CIPHER_CTX *ctx, uint8_t *out,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)